src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -234,7 +235,8 @@ const useCoreChatSystemHook = () => {
     }
 
     refresh('channelList');
-    refresh('chat');
+    refreshWithId('chat', websocketData?.channel?.id);
+    // refresh('chat');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from a general refresh to a more targeted refresh using an ID is a good improvement for performance. It reduces unnecessary data fetching and processing. However, ensure that `websocketData?.channel?.id` always provides a valid id. If there's a chance it could be undefined or null, you should add error handling or fallback mechanisms.
```javascript
if (websocketData?.channel?.id) {
  refreshWithId('chat', websocketData.channel.id);
} else {
  // Handle error or fallback mechanism here
}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->